.ci/before_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+if [[ -n "${CI_TARGET}" ]]; then
+  exit
+fi
+
+if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
+  brew update
+fi
+pip install --user --upgrade pip
```

Comment:
Since you said it doesn't really matter if this is updated, could you remove this line? or does it work now?